domains/Discussions/Tests/Feature/QuestionsViewTest.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Domains\Discussions\Tests\Feature;
+
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Accounts\Models\User;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Response;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Tests\TestCase;
+
+class QuestionsViewTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private User $user;
+    private Question $question;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user     = UserFactory::new()->create();
+        $this->question = QuestionFactory::new(['author_id' => $this->user->id])->create();
```

Comment:
minor: This project's style does not align `=` nor `=>` signs :)

Reply:
Thanks, phpstorm was no longer supposed to change this. Changed!